### DISCUSSION TITLE
GNSS Multipath Analysis Software v.1.2.0

### DISCUSSION THREAD

**Discussion by paarnes** [paarnes/GNSS_Multipath_Analysis_Software#8] [Q&A]
Fixes in v.1.2.0:
* Major improvments of the Rinex v2.xx reading routine (more robust) 
* Add support for Signal To Noise Ratio (SNR) in the software
* Added plot that shows both SNR wrt time and elevation angle
* Added polarplot of SNR
* Latex no longer required to run the software (but its recommended to install Latex to make the text formatting in the plots prettier)
* Made barplot for systems with less than two obscodes prettier (not to thick bars) 
* Fix a small bug in writeoutputfile for GLONASS
* Bug fix in the computeSatElevation angle. In previous releases angles got computed for satellite that was not visible in the current epoch. This is now fixed. 
* Fix float formatting on colorbar in polar plot (now always only 1 decimal) 

In this release, among other things, these issues are fixed: #3 and #6  

<hr /><em>This discussion was created from the release <a href='https://github.com/paarnes/GNSS_Multipath_Analysis_Software/releases/tag/v.1.2.0'>GNSS Multipath Analysis Software v.1.2.0</a>.</em>

**Comment by lz8245** [comment]
"Why is there a significant discrepancy between the multipath error calculation results of this software and those of RTKLIB?"

**Comment by paarnes** [comment]
Hi,

There should not be a substantial difference in the multipath estimates. However, one can see some discrepancy in the RMSE values due to differences in the number of detected cycle slips and so on. For instance, an estimate of the multipath effect on C2X could be based on a linear combination of different signals in this software compared to RTKLIB, which can lead to different statistics.

Could you provide some more information, preferably a test dataset that shows these discrepancies?

Best regards,
Per Helge
